src/instructlab/sysinfo.py

```diff
@@ -69,6 +69,42 @@ def _torch_cuda_info() -> typing.Dict[str, typing.Any]:
     return info
 
 
+def _torch_hpu_info() -> typing.Dict[str, typing.Any]:
+    """Intel Gaudi (HPU) devices"""
+    # pylint: disable=import-outside-toplevel
+    # Third Party
+    import torch
```

Comment:
Can we handle if the import failed and torch dep has not been pulled?

Reply:
InstructLab has a hard requirement on PyTorch. You cannot install InstructLab without PyTorch.